tests/conftest.py

```diff
@@ -48,12 +51,26 @@ def client_with_write_permission(client: TestClient) -> TestClient:
 
 
 @pytest.fixture
-def client_with_read_permission(client: TestClient) -> TestClient:
-    """Return a fastAPI test client with read permission initialized with our app."""
+def client_with_api_key_read_permission(client: TestClient) -> TestClient:
+    """Return a fastAPI test client with read permission granted by API key."""
     client.headers.update({"X-API-Key": "read_key"})
     return client
 
 
+@pytest.fixture
+def client_with_basic_auth_read_permission(client: TestClient) -> TestClient:
+    """Return a fastAPI test client with read permission granted by basic auth."""
+    client.headers.update({"Authorization": "Reader read_password"})
```

Comment:
hm, basic auth headers should rather look like `"basic " + b64encode("Reader:read_password")`. not sure why this works though

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization#basic_authentication

Reply:
I actually wasn't working before, the test client is not used in any of the `test_main.py` atm. We are always using the test client with the API-key authorization method. I fixed it according to your suggestion now and swapped the client in a test locally and it works like this: https://github.com/robert-koch-institut/mex-backend/pull/18/commits/039405b15b3aceabd8dd9156d2d9d10ef1cf35c1